image-loader/app/lib/imaging/FileMetadataReader.scala

```diff
@@ -56,7 +58,10 @@ object FileMetadataReader {
     }
     yield getMetadataWithICPTCHeaders(metadata, imageId) // FIXME: JPEG, JFIF, Photoshop, GPS, File
```

Comment:
Looking at other functions that have been renamed in this PR, should this also be changed to `getMetadataWithIPTCHeaders`?

Reply:
haha yes, someone knew how to start writing IPTC but didn't know how to stop

Reply:
Done.